prow/plugins/plugins.go

```diff
@@ -337,12 +339,21 @@ type Slack struct {
 	MergeWarnings   []MergeWarning `json:"mergewarnings,omitempty"`
 }
 
+// ConfigMapSpec contains configuration options for the configMap being updated by the ConfigUpdater plugin
+type ConfigMapSpec struct {
+	// Name of ConfigMap
+	Name string `json:"name"`
+	// Namespace in which the configMap needs to be deployed. If no namespace is specified
+	// it will be deployed to the namespace Prow is deployed to.
```

Comment:
This is not accurate. It is deployed to the default namespace of the `kube.Client` used by the plugin which happens to be the `ProwJobNamespace`.

Reply:
done